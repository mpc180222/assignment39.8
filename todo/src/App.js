import logo from './logo.svg';
import './App.css';
import React from 'react';
import TodoList from './TodoList';

function App() {
  return (
    <div className="">
      <TodoList />
    </div>
  );
}

export default App;
