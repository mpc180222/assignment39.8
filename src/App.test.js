import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import App from './App';

it('renders without crashing', () => {
  render(<App />);
  
});

it("should match snapshot", () => {
  const { asFragment } = render(<App />)
  expect(asFragment()).toMatchSnapshot();
})
