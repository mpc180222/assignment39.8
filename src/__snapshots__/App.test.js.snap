// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class=""
  >
    <div>
      <div>
        <div
          data-testid="blue"
          style="width: 100px; height: 100px; background-color: blue;"
        />
        <button
          data-testid="remove-blue"
          id="3817f664-ffb9-47ec-86cf-cb94673e92d0"
        >
          X
        </button>
      </div>
      <div>
        <div
          data-testid="red"
          style="width: 100px; height: 100px; background-color: red;"
        />
        <button
          data-testid="remove-red"
          id="6862815e-bf31-455e-ab9a-0e9c7f0aec57"
        >
          X
        </button>
      </div>
      <form>
        <input
          data-testid="new-box-width-input"
          id="width"
          name="width"
          placeholder="width"
          type="text"
          value=""
        />
        <input
          data-testid="new-box-height-input"
          id="height"
          name="height"
          placeholder="height"
          type="text"
          value=""
        />
        <input
          data-testid="new-box-color-input"
          id="color"
          name="color"
          placeholder="color"
          type="text"
          value=""
        />
        <button
          data-testid="new-box-submit-btn"
        >
          Add box
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
